rpc_jobs/standard_job_postmerge.yml

```diff
@@ -0,0 +1,141 @@
+    image:      "xenial"
+    scenario:   "functional"
+    action:     "test"
+    jobs:
+      - 'PM_{name}-{branch}-{image}-{scenario}-{action}'
+
+- job-template:
+    name: 'PM_{name}-{branch}-{image}-{scenario}-{action}'
+    project-type: pipeline
+    concurrent: true
+    FLAVOR: "performance1-1"
+    IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
+    properties:
+      - build-discarder:
+          num-to-keep: "30"
+      - github:
+          url: "{repo_url}"
+    parameters:
+      - rpc_gating_params
+      - instance_params:
+          IMAGE: "{IMAGE}"
+          FLAVOR: "{FLAVOR}"
+          REGIONS: "{REGIONS}"
+          FALLBACK_REGIONS: "{FALLBACK_REGIONS}"
+      - string:
+          name: REPO_URL
+          default: "{repo_url}"
+          description: Url of the repo under test
+      - string:
+          name: BRANCH
+          default: "{branch}"
+          description: Branch of the repo under test
+      - string:
+          name: STAGES
+          default: >-
+            Allocate Resources,
+            Connect Slave,
+            Cleanup,
+            Destroy Slave
+          description: |
+            Pipeline stages to run CSV. Note that this list does not influence execution order.
+            Options:
+              Allocate Resources
+              Connect Slave
+              Cleanup
+              Destroy Slave
+    triggers:
+      - timed: "@daily"
+      - github
+
+    dsl: |
+      library "rpc-gating@${{RPC_GATING_BRANCH}}"
+      properties([pipelineTriggers([githubPush()])])
+
+      // Pass details about the job parameters through
+      // to the target environment so that scripts can
+      // use them to adapt behaviour.
+      env.RE_JOB_NAME = "{name}"
+      env.RE_JOB_IMAGE = "{image}"
+      env.RE_JOB_SCENARIO = "{scenario}"
+      env.RE_JOB_ACTION = "{action}"
+      env.RE_JOB_FLAVOR = "{FLAVOR}"
+      env.RE_JOB_TRIGGER = "periodic"
+
+      // Apply a global three hour timeout
+      timeout(time: 3, unit: 'HOURS'){{
+        common.shared_slave() {{
+          pubcloud.runonpubcloud {{
+            // Set the default environment variables used
+            // by the artifact and test result collection.
+            env.RE_HOOK_ARTIFACT_DIR="${{env.WORKSPACE}}/artifacts"
+            env.RE_HOOK_RESULT_DIR="${{env.WORKSPACE}}/results"
+
+            // Set the job result default
+            currentBuild.result="SUCCESS"
+
+            try {{
+              ansiColor('xterm') {{
+                dir("${{env.WORKSPACE}}/${{env.ghprbGhRepository}}") {{
+                  withCredentials(common.get_cloud_creds()) {{
+
+                    stage('Checkout') {{
+                      git branch: env.BRANCH, url: env.REPO_URL
+                    }} // stage
+
+                    stage('Execute Pre Script') {{
+                      // Retry the 'pre' stage 3 times. The 'post' stage is considered
+                      // to be postparation for the test, so let's try and make sure
```

Comment:
This line and the one above appear to have been changed by mistake from what is written in the pre-merge template. The use of `"post"` instead of `"pre"` doesn't seem to make much sense.

Reply:
Haha postparation, fixed.